crypto/mem_clr.c

```diff
@@ -60,6 +60,7 @@
 #include <string.h>
 #include <openssl/crypto.h>
 
+extern unsigned char cleanse_ctr;
```

Comment:
I don't understand why would that be needed. And I can't reproduce problem with compilers I have... Which one is complaining and what about?